### PR TITLE
PhoneVerificationScreen/UI

### PR DESCRIPTION
**Pull request by vaishnavi-agr**
Designed the Phone Verification Screen
WhatsApp Image 2024-06-21 at 1 27 08 AM

### CODE REVIEW

File: src/UI/Screens/PhoneVerificScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+import {View, Text, TextInput, Image, TouchableOpacity} from 'react-native';
+import styles from './PhoneVerific.styles';
+
+export default function Phone_Verification() {
```
Comment (Jha2508) [inline src/UI/Screens/PhoneVerificScreen.tsx]:
```suggestion
export default function PhoneVerificationScreen() {
```

File: src/UI/Screens/PhoneVerific.styles.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  description: {
+    fontFamily:'Open Sans',
+    fontSize: 12.5,
+    fontWeight: '400',
+    lineHeight: 18,
+    width: 325,
+    left: 5,
+    bottom:-5,
+    marginBottom: 20,
+    // alignItems:'center',
```
Comment (Jha2508) [inline src/UI/Screens/PhoneVerific.styles.ts]: remove comments if not needed

Comment (vaishnavi-agr) [reply src/UI/Screens/PhoneVerific.styles.ts]: Done

File: src/UI/Screens/PhoneVerificScreen.tsx (hunk 1 above)
Comment (vaishnavi-agr) [reply src/UI/Screens/PhoneVerificScreen.tsx]: Done

**Review by Armaggeddon12 — APPROVED** [review]

**Pull request merged**